scRNA-seq/02-tag-based_pre-processing_scRNA-seq.md

```diff
@@ -0,0 +1,152 @@
+# Pre-processing single-cell RNA-seq data 
+
+**CCDL 2019**
+
+#### In this section, we will be running through the basics of pre-processing
+single-cell RNA-seq data.
+
+We will be using a peripheral blood mononuclear cells (pbmc's) 10X Genomics [(Zheng et al, 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601) dataset as an example.
+For 10X Genomics scRNA-seq data, cells are separated by emulsion/droplets, and 
+individual cells are given barcodes.
+These data also have
+[Unique Molecular Identifiers (UMIs)](http://www.nature.com/doifinder/10.1038/nmeth.2772)
+which allow us to better control for PCR amplification errors and biases.
+
+Note: Raw single-cell RNA-seq data from non-tag-based methods, like the Smart-seq2 
+dataset we were working with in the previous section, can be processed using 
+Salmon, just like was done in the bulk RNA-seq module. 
+
+## Steps for Processing scRNA-seq Data:
+
+### Step 0: Obtaining the data
+
+10X Genomics has plenty of [datasets](https://support.10xgenomics.com/single-cell-gene-expression/datasets)
+available to play around with.
+For this example, we will use the [1k pbmc](https://support.10xgenomics.com/single-cell-gene-expression/datasets/3.0.0/pbmc_1k_v2).
+We will process a fastq file of this as an example.  
+For practical purposes as far as time constraints, we have subset this fastq file
+to the first 3 million reads.
+
+*Note*: depending on the state of the data you are working with, ie. if you have
+a `.bcl` file, you will need to use `CellRanger` with their `mkfastq` command to
+make fastq files from this.
+However, because most publicly available data is in fastq format and any
```

Comment:
However, most public data is available in fastq format and a core facility may return your data to you in this format.
We will start from fastq files.